.github/workflows/tests.yaml

```diff
@@ -86,10 +90,10 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [macos-latest]
-        python-version: ['3.9', '3.10']
```

Comment:
I assume that there should not be any python specific issues on OS X. The different Python versions are already tested with the Docker tests.